getting-started/README.md

```diff
@@ -66,33 +68,33 @@ Now we can start the services
 ```
 docker compose up -d
 ```
-It will automatically download the docker image on docker hub:
+It will automatically download the needed docker images from docker hub:
+
 - docker pull opea/chatqna:latest
 - docker pull opea/chatqna-ui:latest
 
-In following cases, you will need to build docker image from source by yourself.
+In the following cases, you will need to build docker image from source by yourself.
 
-1. Failed to download the docker image.
-2. Use the latest or special version.
+- The docker image failed to download.
+- You want to use a different version than latest.
 
-Please refer to the ['Build Docker Images'](/examples/ChatQnA/deploy) section from the file that matches your hardware.
+Refer to the {ref}`ChatQnA Example Deployment Options <chatqna-example-deployment>` section for building from source instructions matching your hardware.
 
-### Interact with ChatQnA Megaservice and Microservice 
+## Interact with ChatQnA Megaservice and Microservice
 ```
 curl http://${host_ip}:8888/v1/chatqna \
     -H "Content-Type: application/json" \
     -d '{
         "messages": "What is the revenue of Nike in 2023?"
     }'
 ```
-This command will provide the response as a stream of text. You can modify the message parameter in the curl command and interact with the ChatQnA service.
+This command will provide the response as a stream of text. You can modify the `message` parameter in the `curl` command and interact with the ChatQnA service.
+
+## What’s Next

```

Comment:
what do we want to communicate in "what's next"? Is it customizing a GenAI pipeline starting with the samples? Why would I want to build a microservice it is pre-built and available on Docker Hub? For supply chain safety. Or to contribute  a new microservice component.

Reply:
I asked @srinarayan-srikanthan to add a concluding section that provided where the reader should go next. @tomlenth @preethivenkatesh Do any of you have a better suggest for a "What's Next" call-to-action section than what @srinarayan-srikanthan provided?

Reply:
@mkbhanda Would it be OK to approve this PR and @tomlenth @preethivenkatesh and @srinarayan-srikanthan can follow up with another PR?  (Tom already mentioned we'll want to do another pass on this.)